### ISSUE TITLE
[Enhancement] Infer `dapr.yaml` when omitted from configuration

### ISSUE THREAD

**Issue by philliphoff** [microsoft/vscode-dapr#284]
If a Dapr task definition or debug launch configuration has been defined without an explicit run file, infer the `dapr.yaml` at the root of the workspace (if present).  This will make creation/scaffolding of such configuration files simpler.

**Comment by IvanJobs** [comment]
@MregXN You can investigate on this one first.

**Comment by MregXN** [comment]
Hi, @philliphoff. Is that mean the local dapr.yaml file should be read by this extension and properties in this file needs to be filled in task.json and launch.json?

**Comment by philliphoff** [comment]
No, the suggestion is that, if the `tasks.json` and `launch.json` files omit the property that would normally point to the one and only `dapr.yaml` in the repo, the extension, when running the task and/or launching the debugger would simply assume that that `dapr.yaml` is the one to be used in the `dapr run -f ...` command.  The `.json` files need not be changed.  We're just saying that the user doesn't need to tell the extension (in the `.json` files) which file to use because the extension can figure it out *at the time it actually needs to use it*.

**Comment by MregXN** [comment]
Hi, @philliphoff. I wanna confirm my understanding is correct with following code snippet. 

tasks.json:
```json
      {
          "label": "dapr",
          "type": "dapr",
         // "runFile": "./dapr.yaml"
      }
```
launch.json:
```json
      {
          "name": "Launch Dapr",
          "request": "launch",
          "type": "dapr",
          //"runFile": "${workspaceFolder}/dapr.yaml",
          "preLaunchTask": "dapr"
      }
```

If users do not specify a runFile (e.g., it is commented out in the code snippet), but task and launch configurations of type "dapr" still exist, extension should automatically use the dapr.yaml in the workspace without modifying these two json files. Is this right?

**Comment by philliphoff** [comment]
Closed with #300.
